pywbem_mock/_wbemconnection_mock.py

```diff
@@ -128,6 +134,51 @@ def __repr__(self):
             "registry={s._registry}, ",
             s=self)
 
+    def display_registered_providers(self, dest=None):
+        """
+        Generate display of registry in readable form and return the string
+        Output format is:
+        <namespace>:
+          <classname>: CIM_Namespace provider: NamespaceProvider type: instance
+
+        For example:
+        root/cimv2:
+          CIM_Namespace, provider: NamespaceProvider type: instance
+          CIM_Blah, provider: BlahProvider, type: instance
+
```

Comment:
Reading the code (I did not try it out), the output is for example:
 `CIM_Blah  BlahProvider  instance`
Ie not including keywords like provider or type.

Reply:
I modified the comment to match the current code. Fixed example also.